includes/notes/class-wc-stripe-whats-new-note.php

```
@@ -0,0 +1,149 @@
+	 */
+	public static function is_upgrade_pending(): bool {
+		$last_seen = get_option( self::LAST_SEEN_VERSION_OPTION );
+
+		// No baseline yet: can't tell an upgrade from a fresh install, so stay
+		// silent until record_install_version() seeds it.
+		if ( false === $last_seen || '' === $last_seen ) {
+			return false;
+		}
+
+		return version_compare( $last_seen, WC_STRIPE_VERSION, '<' );
+	}
```

PHPMD (2.15.0)

[error] The method is_upgrade_pending is not named in camelCase. (undefined)

(CamelCaseMethodName)